modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -314,4 +317,8 @@ export class StakingWallet implements IStakingWallet {
       return true;
     }
   }
+
+  private async getDescriptorWallet(transaction: StakingTransaction): Promise<IWallet> {
+    return await this.wallet.baseCoin.wallets().get({ id: transaction.buildParams?.senderWalletId });
+  }
```

Comment:
are there any tests you could update/add for this?

Reply:
good question, but I cannot find any test in sdk-core for wallet/stakingWallet at all.

Reply:
Good chance to add some!